src/components/views/settings/encryption/ResetIdentityPanel.tsx

```diff
@@ -78,18 +82,29 @@ export function ResetIdentityPanel({ onCancelClick, onFinish, variant }: ResetId
                 <EncryptionCardButtons>
                     <Button
                         destructive={true}
+                        disabled={inProgress}
                         onClick={async (evt) => {
+                            setInProgress(true);
                             await matrixClient
                                 .getCrypto()
                                 ?.resetEncryption((makeRequest) => uiAuthCallback(matrixClient, makeRequest));
                             onFinish(evt);
                         }}
                     >
+                        {inProgress && <InlineSpinner />}
                         {_t("action|continue")}
                     </Button>
-                    <Button kind="tertiary" onClick={onCancelClick}>
-                        {_t("action|cancel")}
-                    </Button>
+                    {inProgress ? (
+                        <EncryptionCardEmphasisedContent>
+                            <span className="text-warning">
```

Comment:
Can we use compound variable here instead of global `test-warning` which is relying on `$alert`

Reply:
Sorry, I don't know understand exactly what you mean here.  Should I be creating a new style and use `var(--cpd-color-text-critical-primary)`?  Or is there a different class or element that I should be using?

Reply:
b77591f3e0 creates a new CSS class which uses the compound variable -- hopefully that is what was intended.